ci/install-bazel-rbe.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/bash
 
 set -e
+
+function install_rpmbuild() {
```

Comment:
Let's call the function `install_dependencies` instead. Because it reads better from a high level

Reply:
@lolski fixed in https://github.com/graknlabs/build-tools/pull/78/commits/1c3dc1b28ed7a6c165806bc4d24e1dc493c6ecfa